bokehjs/src/lib/models/dom/index.ts

```diff
@@ -105,8 +108,11 @@ export abstract class Placeholder extends DOMNode {
 export class IndexView extends PlaceholderView {
   override model: Index
 
-  update(_source: ColumnarDataSource, i: DataIndex, _vars: object/*, formatters?: Formatters*/): void {
-    this.el.textContent = i.toString()
+  update(_source: ColumnarDataSource, i: DataIndex | null, _vars: object/*, formatters?: Formatters*/): void {
+    if (i == null)
+      this.el.textContent = "(null)"
+    else
+      this.el.textContent = i.toString()
```

Comment:
Preferably, as in other implementations:
```suggestion
  update(_source: ColumnarDataSource, i: DataIndex | null, _vars: object/*, formatters?: Formatters*/): void {
    this.el.textContent = i == null ? "(null)" : i.toString()
```
Not to make code shorter, but to indicate that it's text that varies with `i`, not the target of the assignment.